docs/reference/mapping/types/keyword.asciidoc

```diff
@@ -1,20 +1,30 @@
+[testenv="basic"]
 [[keyword]]
-=== Keyword field type
+=== Keyword type family
 ++++
 <titleabbrev>Keyword</titleabbrev>
 ++++
 
-A field to index structured content such as IDs, email addresses, hostnames,
-status codes, zip codes or tags.
+The keyword family includes the following field types:
+
+* <<keyword,`keyword`>>, which is used for structured content such as IDs, email
+addresses, hostnames, status codes, zip codes, or tags. 
+* <<constant-keyword,`constant_keyword`>> for keyword fields that always contain
+the same value.
+* <<wildcard,`wildcard`>>, which optimizes log lines and similar keyword values
+for grep-like <<query-dsl-wildcard-query,wildcard queries>>.
 
-They are typically used for filtering (_Find me all blog posts where
-++status++ is ++published++_), for sorting, and for aggregations. Keyword
-fields are only searchable by their exact value.
+Keyword fields are often used in <<query-dsl-bool-query,search filters>>,
```

Comment:
As a new user, it seems hard to tell how keyword fields relate to the `bool` filter option. It could be clearer to say something like "Keyword fields are often used in term-level queries like `term`" and link to our 'Term-level queries' docs?

Reply:
Good idea! Thanks!